back/app/models/custom_page.rb

```diff
@@ -49,6 +51,8 @@ class CustomPage < ApplicationRecord
 
   validates :slug, presence: true, uniqueness: true
 
+  validates :code, inclusion: %w[custom]
```

Comment:
`StaticPage` has
```
CODES = %w[about terms-and-conditions privacy-policy faq proposals custom].freeze
...
validates :code, inclusion: { in: CODES }
validates :code, uniqueness: true, if: ->(page) { !page.custom? }
...
def custom?
  code == 'custom'
end
```
Should we include any or all of these, rather than my extremely simple validation?

Reply:
Ah and uniqueness validation too. Like
```
validates uniqueness: true, unless: :custom?
```

Reply:
> As we want to preserve the old behavior (not allow removing default pages), I think we should keep also about, faq, proposals codes
> 
> https://github.com/CitizenLabDotCo/citizenlab/blob/06649f3dfc190d08235cdda69f4afb3c7745e48a/front/app/containers/Admin/pages/All/index.tsx#L47

@alexander-cit But aren't `terms-and-conditions privacy-policy` both "default pages" also?

Reply:
Oh, I see, I think. you mean:
`about`, `faq`, `proposals`, and `custom` are deletable, but `terms-and-conditions` & `privacy-policy` are not? (hence the latter should not be in the array).

Reply:
terms-and-conditions & privacy-policy will stay in the `static_pages` table. So, we don't need them here.